src/frontend/src/components/Post/Post.jsx

```diff
@@ -92,9 +100,9 @@ const Post = ({ postUrl }) => {
   const sectionEl = useRef(null);
   // Grab the post data from our backend so we can render it
   const { data: post, error } = useSWR(postUrl, (url) => fetch(url).then((r) => r.json()));
-  const [expandToggle, setToggle] = React.useState(true);
-  const expandTitleOrNot = () => {
-    setToggle(!expandToggle);
+  const [expandHeader, setExpandHeader] = useState(true);
```

Comment:
Probably we want `false` as the default, no?  That is, we don't want this to always start expanded.